README.md

```diff
@@ -50,22 +50,24 @@ The project is delivered as a docker image that contains our tooling:
 docker pull asecurityteam/sdcli:v1
 ```
 
-With the image installed you call it like:
+With the image installed you call it like (omit the first `--mount` if on Mac):
 
 ```bash
+export cwd=$(pwd)
+export project_path=${cwd#"${GOPATH}/src/"}
 docker run -ti \
-    # Mount and configure SSH inside the container.
+    # If Linux, mount and configure SSH inside the container.
     --mount src=${SSH_AUTH_SOCK},target=/ssh-agent,type=bind \
     --env SSH_AUTH_SOCK=/ssh-agent \
     # Mount the current project directory to a patch inside the container.
-    --mount src="$(pwd)",target=/go/src/github.com/asecurityteam/go-vpcflow,type=bind \
+    -v "$(pwd -L):/go/src/${project_path}" \
```

Comment:
I think we should stick with the `--mount` syntax here for consistency, i.e.

`--mount src="$(pwd -L)",target="/go/src/${project_path}",type=bind`

Reply:
I've updated a few spots for `--mount` consistency and use of double-quote consistency

Reply:
The double quotes are recommended in bash to avoid unintentional splitting. For example: https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
Yup, especially necessary here due to paths possibly having spaces or special chars

Reply:
To clarify, I meant to ask why you removed double quotes from other var usage. Running the example through ShellCheck shows several cases where double quotes aren't used but likely should be.